content/miq/dialogs/volume.yaml

```diff
@@ -0,0 +1,247 @@
+---
```

Comment:
If you put this in `content/miq_dialogs` it will be automatically seeded for you.  Also since these are seeded into the global namespace please include the operation and provider name like `cloud_volume_autosde_order.yml`

Reply:
where the dialog will be automatically seeded?

Reply:
Dialogs are seeded here https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_dialog/seeding.rb this will automatically create MiqDialog database records that the user can select